components/Cards/BUILD

```diff
@@ -28,19 +28,66 @@ load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
 
 licenses(["notice"])  # Apache 2.0
 
-mdc_public_objc_library(
+mdc_objc_library(
     name = "Cards",
+    srcs =
+        native.glob(
+            ["src/*.m"],
+            exclude = [
+                "src/MDCCard+Ripple.h", 
+                "src/MDCCardCollectionCell+Ripple.h",
+                "src/MDCCard+Ripple.m", 
+                "src/MDCCardCollectionCell+Ripple.m",
+            ],
+        ),
+    hdrs =
+        native.glob(
+            ["src/*.h"],
+            exclude = [
+                "src/MDCCard+Ripple.h",
+                "src/MDCCardCollectionCell+Ripple.h",
+            ],
+        ),
+    enable_modules = 1,
+    includes = ["src"],
     sdk_frameworks = [
         "CoreGraphics",
         "UIKit",
     ],
+    visibility = ["//visibility:public"],
     deps = [
         "//components/Ink",
         "//components/ShadowLayer",
         "//components/ShapeLibrary",
         "//components/Shapes",
         "//components/private/Icons/icons/ic_check_circle",
         "//components/private/Math",
+        ":CardsPrivate"
+    ],
+)
+
+mdc_objc_library(
+    name = "CardsPrivate",
+    sdk_frameworks = [
+        "UIKit",
+    ],
+    hdrs = native.glob(["src/private/*.h"]),
+    includes = ["src/private"],
+    visibility = ["//visibility:public"],
+)
+
+mdc_objc_library(
```

Comment:
Consider annotating this as a Beta component with a comment.

Reply:
Done.